spring-data-rest/src/main/java/com/baeldung/springdatawebsupport/application/repositories/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.springdatawebsupport.application.repositories;
+
+import com.baeldung.springdatawebsupport.application.entities.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.querydsl.QuerydslPredicateExecutor;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends PagingAndSortingRepository<User, Long>,
+        QuerydslPredicateExecutor<User> {
```

Comment:
```suggestion
  QuerydslPredicateExecutor<User> {
```

Reply:
indentation

Reply:
No problem. I fixed the indentation in here as well.